data/roles.js

```diff
@@ -1568,12 +1636,27 @@ const roleData = {
         "Wins if their chosen lovers are alive at the end of the game.",
       ],
     },
+    Angel: {
+      alignment: "Independent",
+      description: [
+        "Will become the guardian angel for one person in the game.",
+        "Will sacrifice himself if the person is in danger.",
+        "Wins if person is alive at the end of the game.",
+      ],
+    },
     Sidekick: {
       alignment: "Independent",
       description: [
           "Assigned to a random independent player at game start.",
           "Wins if their teammate wins.",
       ],
+  },
+  "Ripper": {
+    alignment: "Independent",
+    description: [
+      "Kills one player every night.",
+      "Wins when all other hostile third parties are dead.",
```

Comment:
this implies that ripper can win with village and mafia

Reply:
Yes, that is intentional

lucid made it a hostile third on EM, but that meant it was functionally identical to an SK that can't win in a final two with another hostile third